app/src/main/java/gitlin/kothub/github/LoginWebView.kt

```diff
@@ -4,35 +4,28 @@ import android.app.Activity
 import android.content.Intent
 import android.graphics.Bitmap
 import android.net.Uri
+import android.util.Log
 import android.webkit.WebResourceRequest
+import android.webkit.WebResourceResponse
 import android.webkit.WebView
 import android.webkit.WebViewClient
 
 
 class LoginWebView(val context: LoginWebViewActivity): WebViewClient() {
 
-    // dialog = ProgressDialog(context)
 
-    init {
-        //  dialog.setTitle("Loading...")
-        //dialog.progress = 0
+    override fun shouldOverrideUrlLoading(view: WebView?, url: String?): Boolean {
+        val result = shouldOverrideUrlLoading(view, Uri.parse(url))
+        return if (result) result else  super.shouldOverrideUrlLoading(view, url)
     }
 
-    override fun onPageStarted(view: WebView?, url: String?, favicon: Bitmap?) {
-        //dialog.show()
-    }
+    override fun shouldOverrideUrlLoading(view: WebView?, request: WebResourceRequest?): Boolean {
```

Comment:
`override fun shouldOverrideUrlLoading(view: WebView?, request: WebResourceRequest?)` is defined twice (line 22 and 29)